package.json

```diff
@@ -7,7 +7,8 @@
     "jss": "bin/cli.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "css2jss": "node bin/cli example/style.css",
+    "test": "node example/test"
```

Comment:
Well this is not exactly test, having tests is nice, but this is just a running example. I would suggest to either add real tests or just to remove this line.